packages/connect/src/backend/Blockchain.ts

```diff
@@ -40,6 +40,11 @@ const getNormalizedShortcut = (shortcut: string) => {
         return 'XRP';
     }
 
+    // Can be safely removed when both Polygon PoS Blockbooks return POL as shortcut
+    if (shortcut.toLowerCase() === 'pol') {
```

Comment:
shouldnt this be the other way around? checking 'matic' and returning 'pol' ?

Reply:
Yes, thats why i have todo few lines below

Reply:
3a09c0211db5b258279e59c6d6e4aab89d1f460a this is okay, right?